apps/Better Chromium/uninstall

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+# rpi ubuntu (22.04+) use Pi OS packaged chromium for better hardware support
+purge_packages
+rm_external_repo "chromium" force
```

Comment:
I suggest changing the repo name to something more unique, just in case some user out there has made their own `/etc/apt/sources.list.d/chromium.list`, `/usr/share/keyrings/chromium-archive-keyring.gpg`, or `/etc/apt/sources.list.d/chromium.sources` files.

Reply:
`pi-apps-coders-chromium` sound good or do you have another suggestion?

Reply:
Sounds good.